test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua

```diff
@@ -0,0 +1,166 @@
+-- User story: Smoke
+-- Use case: PerformAudioPassThru
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [PerformAudioPassThru] SUCCESS: getting SUCCESS:UI.PerformAudioPassThru()
+--
+-- Description:
+-- Mobile application sends valid PerformAudioPassThru request and gets UI.PerformAudioPassThru "SUCCESS"
+-- response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests PerformAudioPassThru with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if PerformAudioPassThru is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the UI part of request with allowed parameters to HMI
+-- SDL transfers the TTS part of request with allowed parameters to HMI
+-- SDL receives UI part of response from HMI with "SUCCESS" result code
+-- SDL receives TTS part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+
+--[[ Local Variables ]]
+local requestParams = {
+  initialPrompt = {
+    {
+      text = "Makeyourchoice",
+      type = "TEXT",
+    },
+  },
+  audioPassThruDisplayText1 = "DisplayText1",
+  audioPassThruDisplayText2 = "DisplayText2",
+  samplingRate = "8KHZ",
+  maxDuration = 2000,
+  bitsPerSample = "8_BIT",
+  audioType = "PCM",
+  muteAudio = true
+}
+
+local requestUiParams = {
+  audioPassThruDisplayTexts = { },
+  maxDuration = requestParams.maxDuration,
+  muteAudio = requestParams.muteAudio
+}
+
+requestUiParams.audioPassThruDisplayTexts[1] = {
+  fieldName = "audioPassThruDisplayText1",
+  fieldText = requestParams.audioPassThruDisplayText1
+}
+
+requestUiParams.audioPassThruDisplayTexts[2] = {
+  fieldName = "audioPassThruDisplayText2",
+  fieldText = requestParams.audioPassThruDisplayText2
+}
+
+local requestTtsParams = {}
+requestTtsParams.ttsChunks = commonFunctions:cloneTable(requestParams.initialPrompt)
+requestTtsParams.speakType = "AUDIO_PASS_THRU"
+
+local allParams = {
+  requestParams = requestParams,
+  requestUiParams = requestUiParams,
+  requestTtsParams = requestTtsParams
+}
+
+--[[ Local Functions ]]
+local function file_check(file_name)
+  local file_found = io.open(file_name, "r")
+  if nil == file_found then
+    return false
+  else
```

Comment:
@GetmanetsIrina Else statement can be omitted here.

```
local function file_check(file_name)
  local file_found = io.open(file_name, "r")
  if nil == file_found then
     return false
  end
  return true
end
```

Reply:
@aderiabin , Done in fb183773759e98c5603d803a4be087a44dba7ae6.